app/src/main/java/homer/common/KelvinTemperature.java

```diff
@@ -0,0 +1,29 @@
+package homer.common;
+
+public final class KelvinTemperature implements Temperature {
+
+    final static double DELTA_KELVIN_CELSIUS = 273.15;
+    final static double DELTA_FARENHEIT_KELVIN = 459.67;
```

Comment:
```suggestion
    static final double DELTA_KELVIN_CELSIUS = 273.15;
    static final double DELTA_FARENHEIT_KELVIN = 459.67;
```
final and static should be swapped

Reply:
pretty sure it should be like the original commit, it fixed linter errors. Anyways, is the change itself fine? i'll correct all linter errors for the new classes and then merge.

Reply:
The [java specification](https://docs.oracle.com/javase/specs/jls/se17/html/jls-8.html#jls-8.3.1) suggests the `static final` order as a convention so it would be weird for the linter to say otherwise.